docs/model_library/strategic_water_management/index.rst

```diff
@@ -302,6 +302,8 @@ Strategic Model Mathematical Notation
 
 :math:`\textcolor{green}{σ_{s}^{Processing,Storage}}` =                Weekly processing (e.g. clarification) capacity at storage site
 
+:math:`\textcolor{green}{σ_{n}^{Node}}` =                              Daily capacity per network node
```

Comment:
Because this version of the model is still primarily in bbl/week, I think this should be weekly capacity instead of "daily". This is a good note for me, though-  when I update the unit conversion PR  this should be changed back to daily capacity.